notification/telegram.go

```diff
@@ -369,6 +365,18 @@ func (t telegram) OnOrder(order model.Order) {
 
 func (t telegram) OnError(err error) {
 	title := "🛑 ERROR"
-	message := fmt.Sprintf("%s\n-----\n%s", title, err)
-	t.Notify(message)
+
+	var orderError *exchange.OrderError
+	if errors.As(err, &orderError) {
+		message := fmt.Sprintf(`%s
+		-----
+		Pair: %s
+		Quantity: %.4f
+		-----
+		%s`, title, orderError.Pair, orderError.Quantity, orderError.Err)
+		t.Notify(message)
+		return
+	}
```

Comment:
Hi @RobertKwiatkowski, I included a suggestion with other small ajustments.
What you think about this format?

Reply:
Hi, very nice. Looks good to me!